modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -87,7 +101,7 @@ private Map<String, String> _getMap(String fileName) throws Exception {
 		String[] lines = StringUtil.splitLines(FileUtil.read(importsFile));
 
 		for (String line : lines) {
-			int separatorIndex = line.indexOf("=");
+			int separatorIndex = line.indexOf(StringPool.EQUAL);
```

Comment:
Good job, @michaelmdelcavalcanti! In the next pulls, please try to separate this kind of SF in a separated commit :wink: